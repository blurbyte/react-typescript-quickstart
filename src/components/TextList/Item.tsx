import styled from '@styles/styledComponents';

import Text from '../Text';

const ListItem = Text.withComponent('li');

const Item = styled(ListItem)`
  margin: 0.2rem 0;
`;

export default Item;
