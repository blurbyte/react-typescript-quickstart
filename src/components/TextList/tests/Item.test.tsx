import * as React from 'react';
import { shallow } from 'enzyme';

import Item from '../Item';

describe('Item', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Item />);
    expect(wrapper).toMatchSnapshot();
  });
});
