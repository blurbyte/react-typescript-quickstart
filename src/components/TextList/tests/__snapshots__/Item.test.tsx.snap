// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item renders correctly 1`] = `
.c0 {
  margin: 0.2rem 0;
}

<styled.li
  className="c0"
/>
`;
